### ISSUE TITLE
Consider color scheme options

### ISSUE THREAD

**Issue by lognaturel** [opendatakit/website#44]
Perhaps whites and grays would be more neutral? Brown/tan tends to be polarizing.

**Comment by yanokwa** [comment]
I agree that a more neutral color would be better to start with. I looked at https://github.com/mmistakes/minimal-mistakes#skins-color-variations and it seems the default is, well, a good default. What do you think, @downeym

**Comment by wbrunette** [comment]
Does anybody who is "watching" the website have a preference on colors? Now would be the time to speak up.
